<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>528</width>
    <height>322</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Turing Machine Emulator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="outputLabel">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="inputLineEdit">
      <property name="text">
       <string>Входная лента</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addStateButton">
        <property name="text">
         <string>Добавить состояние</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteStateButton">
        <property name="text">
         <string>Удалить состояние</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QTableWidget" name="tableWidget"/>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="runButton">
      <property name="text">
       <string>Запуск</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>528</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="separator"/>
    <addaction name="newMachine"/>
    <addaction name="openMachine"/>
    <addaction name="saveMachine"/>
    <addaction name="separator"/>
    <addaction name="quit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="newMachine">
   <property name="text">
    <string>Создать</string>
   </property>
  </action>
  <action name="openMachine">
   <property name="text">
    <string>Открыть</string>
   </property>
  </action>
  <action name="saveMachine">
   <property name="text">
    <string>Сохранить</string>
   </property>
  </action>
  <action name="quit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
